config/database.php

```diff
@@ -50,7 +50,7 @@
             'charset' => 'utf8mb4',
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => '',
-            'strict' => true,
```

Comment:
I don't know how this will affect the production deploy. Best to leave it out of this PR